Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -82,10 +83,22 @@ namespace Camera
                     ->ClassElement(AZ::Edit::ClassElements::Group, "Render To Texture")
                     ->DataElement(AZ::Edit::UIHandlers::Default, &CameraComponentConfig::m_renderTextureAsset, "Target texture", "The render target texture which the camera renders to.")
                     ->DataElement(AZ::Edit::UIHandlers::Default, &CameraComponentConfig::m_pipelineTemplate, "Pipeline template", "The root pass template for the camera's render pipeline")
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &CameraComponentConfig::m_allowPipelineChanges, "Allow pipeline changes", "If true, the camera's render pipeline can be changed at runtime.")
+                        ->Attribute(AZ::Edit::Attributes::Visibility, &CameraComponentConfig::GetAllowPipelineChangesVisibility)
                 ;
             }
         }
     }
+    AZ::u32 CameraComponentConfig::GetAllowPipelineChangesVisibility() const
+    {
+        bool experimentalFeaturesEnabled = false;
+        const auto registry = AZ::SettingsRegistry::Get();
+        if (registry)
+        {
+            registry->Get(experimentalFeaturesEnabled, "/O3DE/Atom/ExperimentalFeaturesEnabled");
```

Comment:
Maybe add about this cvar to the wiki as well. Maybe here? - https://github.com/o3de/o3de/wiki/%5BAtom%5D-Scene-and-Render-Pipeline